eng/WpfArcadeSdk/tools/RunDrtsLocal.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <!-- The internal and external repos each define their own drts to add that need running. -->
```

Comment:
I generally like this idea, let me potentially suggest an improvement.

Instead of a complicated target file being the dictator of what gets added, can't we have a simple ItemGroup in a props file?  Then the messier parts of the target (like platform detection, etc) just live in the WPF Arcade SDK.  The individual repos just change up the ItemGroup includes.